### ISSUE TITLE
.kdbx File Import (KeePass)

### ISSUE THREAD

**Issue by 8l4ckSh33p** [fcturner/passwords#52]
Is it possible to import .kdbx?

**Comment by Tramour** [comment]
Very good idea. I use keepass myself.

**Comment by fcturner** [comment]
Nice idea! Can you post the plain content of such a file here?

**Comment by 8l4ckSh33p** [comment]
https://familie-liebig.net/index.php/s/9Z0Efyfkzrgov98

**Comment by fcturner** [comment]
Thanks.

This is a HEX-file, which makes it unreadable, even when converted to string.
Does KeePass has an option to export the passwords in a human readable format?

**Comment by fcturner** [comment]
GOT IT!

Export as `KeePass CSV (1.*)` :

![image](https://cloud.githubusercontent.com/assets/7526559/9112805/4dd8ca0e-3c51-11e5-966c-b159420db1a5.png)

Output of `testfile.csv`:
```csv
"Account","Login Name","Password","Web Site","Comments"
"Sample Entry","User Name","Password","http://keepass.info/","Notes"
"Sample Entry #2","Michael321","12345","http://keepass.info/help/kb/testform.html",""
```

I can make this work, and I think it is a wonderful idea! :smile:
I think I will opt for [JQuery CSV](https://code.google.com/p/jquery-csv/), which will be included in a future release as a `.js`-file.

**Comment by fcturner** [comment]
Implemented as of release [8.0.10](https://github.com/fcturner/passwords/releases).
